packages/now-cli/src/util/dev/builder.ts

```diff
@@ -236,9 +236,25 @@ export async function executeBuild(
     result.routes = normalized.routes || [];
   }
 
+  const { output } = result;
+
+  // Mimic fmeta-util and convert cleanUrls
+  if (nowConfig.cleanUrls) {
+    Object.keys(output)
```

Comment:
Why not `Object.entries()` and then `.map()` below can be avoided?

Reply:
Good point, I changed to `Object.entries()`